plugins/notifications/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@backstage/plugin-notifications-frontend",
+  "name": "@backstage/plugin-notifications",
   "description": "Frontend part of the notifications providing the Notifications API, Notifications page, left-side menu item and other REACT components.",
   "version": "0.1.0",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
-  "private": true,
+  "private": false,
```

Comment:
same as above, we can avoid this over making it false

Reply:
done